### PR TITLE
updated the README for runtimes

### PR DESCRIPTION
**Pull request by AndreiNavoichyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,23 +72,28 @@ $ ls ./my-chain-state/chains/dev
...
 #### Runtime `cere`
 
-Runtime `cere` uses by default in Cere Mainnet/Testnet/QAnet. You can start the node with it by running the command:
-```bash
-./target/release/cere --chain=./target/release/customSpecRaw.json
-```
+Runtime `cere` uses by default in Cere Mainnet/Testnet/QAnet. You can start the node with it by:
+1. Running the node connected to [Cere Mainnet](#mainnet), [Cere Testnet](#testnet) or [Cere QAnet](#qanet)
+2. Running the node with a custom spec. Be sure that [id](https://github.com/Cerebellum-Network/blockchain-node/blob/dev-cere/node/service/src/chain_spec.rs#L265) **does not** start with `cere-dev`
+    ```bash
```
Comment (yahortsaryk) [inline README.md:84]: The prefix for the id field is `cere_dev` and not `cere-dev`

File: README.md [hunk 2]
```diff
@@ -72,23 +72,28 @@ $ ls ./my-chain-state/chains/dev
...
 
-Runtime `cere-dev` uses by default in Cere Devnet. You can start the node with it by running the command:
-```bash
-./target/release/cere --chain=./target/release/customSpecRaw.json --force-cere-dev
-```
+Runtime `cere-dev` uses by default in Cere Devnet. You can start the node with it by:
+1. Running the node connected to [Cere Devnet](#Devnet)
+1. Running the [Single-Node Development Chain](#Single-Node-Development-Chain)
+1. Running the node with a custom spec. Be sure that [id](https://github.com/Cerebellum-Network/blockchain-node/blob/dev-cere/node/service/src/chain_spec.rs#L265) **starts** with `cere-dev` and you pass `--force-cere-dev` parameter
+    ```bash
```
Comment (yahortsaryk) [inline README.md:94]: The prefix of id field we are talking about is `cere_dev` and not `cere-dev`

**Review by yahortsaryk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AndreiNavoichyk) [reply README.md:84]: Fixed

File: README.md (hunk 2 above)
Comment (AndreiNavoichyk) [reply README.md:94]: Fixed

**Review by yahortsaryk — APPROVED** [review]

**Pull request merged**